typing/parmatch.ml

```diff
@@ -1558,21 +1558,14 @@ and push_no_or_column rs = List.map push_no_or rs
 (* Those are adaptations of the previous homonymous functions that
```

Comment:
That comment should be updated (or simply removed?), you're only "adapting" one function, and its name is different.